javascript/javascript2/week3/mylocation.js

```diff
@@ -0,0 +1,18 @@
+const button = document.querySelector('button');
+
+const myGeoLocation = () => {
+	if (navigator.geolocation) {
+		navigator.geolocation.getCurrentPosition(success);
+	} else {
+		return 'Not found';
+	}
+};
+
+function success(position) {
```

Comment:
Try to put better naming
```suggestion
function printLocation(position) {
```